ReactiveCocoaTests/Swift/SignalProducerSpec.swift

```diff
@@ -233,46 +233,119 @@ class SignalProducerSpec: QuickSpec {
 		}
 
 		describe("first") {
-			pending("should start a signal then block on the first value") {
+			it("should start a signal then block on the first value") {
+				let (producer, sink) = SignalProducer<Int, NoError>.buffer()
+
+				var result: Result<Int, NoError>?
+				dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0)) {
```

Comment:
Yeah, that's fair. Can we make this more deterministic, though? Maybe by using a dispatch group that we wait upon?